src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1470,7 +1470,12 @@ def __getattr__(self, name: str):
                 )
             return alias_obj
         else:
-            return getattr(super(), name)
+            try:
+                return getattr(super(), name)
+            except AttributeError as ex:
+                raise AttributeError(
+                    self.__class__.__name__ + " has no attribute " + name
+                ) from ex
```

Comment:
Do we need the traceback or shall we raise it from None?

Reply:
Yes, we need it, it helps to identify the exact cause.

Reply:
> Yes, we need it, it helps to identify the exact cause. This change has been done to give more informative message only.
Yes, @hpohekar makes a good point. Transparency is generally important in error messages.

Reply:
yes, that's good as well.

Reply:
@mkundu1 Do you mean like this ? I think this a hacky approach.

```
try:
    getattr(super(), name)
except AttributeError as ex:
    ex.args = (f"'{self.__class__.__name__}' has no attribute '{name}'",)
    raise
```

Output - 

```
solver.setup.named_expressions.child_names
Traceback (most recent call last):
  File "d:\Repos\pyfluent\debug.py", line 323, in <module>
    solver.setup.named_expressions.child_names
  File "D:\Repos\pyfluent\src\ansys\fluent\core\solver\flobject.py", line 1474, in __getattr__
    return getattr(super(), name)
AttributeError: 'named_expressions' has no attribute 'child_names'
```